lib/ex_unit/lib/ex_unit.ex

```diff
@@ -299,7 +299,7 @@ defmodule ExUnit do
 
   defp put_seed(opts) do
     Keyword.put_new_lazy(opts, :seed, fn ->
-      :os.timestamp |> elem(2)
+      rem :os.system_time, 1_000_000
```

Comment:
This is clever to use the native units so we always get the most precision for the OS. Smart enough that we should have a comment, otherwise someone could add the `:microsecond` unit, and revert this fix. Also lets use `System.system_time()` so that a user reading the code has better access to docs.